4-perftools/Dockerfile.rhel7

```diff
@@ -38,7 +37,7 @@ USER 0
 
 WORKDIR ${HOME}
 
-COPY contrib/etc/scl_enable /root/etc/scl_enable
+COPY root/root/etc/scl_enable /root/etc/scl_enable
```

Comment:
I also believe it's not necessary (though it's after the `USER 0` declaration which might affect it?), but I think that it should be removed as part of a different PR so as not to muddy the history.